ledger/artifactmanager/handler.go

```diff
@@ -65,6 +66,7 @@ func (h *MessageHandler) Link(components core.Components) error {
 	h.jetDropHandlers[core.TypeGetObject] = h.handleGetObject
 	h.jetDropHandlers[core.TypeGetDelegate] = h.handleGetDelegate
 	h.jetDropHandlers[core.TypeGetChildren] = h.handleGetChildren
+	h.jetDropHandlers[core.TypeGetHistory] = h.handleGetHistory
```

Comment:
Why `GetHistory` is involved in JetDrop?

Reply:
Yes, let's remove it from jetDrop handlers.